### PR TITLE
Issue #191: SplitImage basic unit tests

### PR DESCRIPTION
**Pull request by vatsalj**
I've not added unit tests for checking keys in localStorage yet (as written in Messages tests). I've done so because keys are saved to localStorage in Messages but not in SplitImage. So do you want me to add the feature to save keys in localStorage? Tests for the same can be written then.

### CODE REVIEW

**Comment by vatsalj** [comment]
Issue #191 : I've not built the templates now (as they were not built by @smcgregor initially). Do let me know if that's required.

File: SplitImage/test/new.js [hunk 1]
```diff
@@ -5,9 +5,43 @@
  
 describe ("SplitImage New Suite", function() {
   
+  // Get an HTML document defined by the pre-processor.
+  // This is a rough hack because HTML2JS seems to assign the
+  // key to the absolute URL, which is not reliable on
+  // continuous integration.
+  beforeEach(function() {
+    var keys = Object.keys(__html__);
```
Comment (smcgregor) [inline SplitImage/test/new.js]: You can't use this hack anymore. You must create the DOM elements you need to test inside the javascript.

File: SplitImage/test/show.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
-describe ("SplitImaage Show Suite", function() {
-  
+describe ("SplitImage Show Suite", function() {
+
+  // Get an HTML document defined by the pre-processor.
+  // This is a rough hack because HTML2JS seems to assign the
+  // key to the absolute URL, which is not reliable on
+  // continuous integration.
+  beforeEach(function() {
+    var keys = Object.keys(__html__);
```
Comment (smcgregor) [inline SplitImage/test/show.js]: Don't use the HTML creation hack. Create the DOM nodes in JS.

**Comment by smcgregor** [comment]
Can you please address the comments I left? Also, this PR includes updating the branch from master. These should be two separate pull requests.

**Comment by vatsalj** [comment]
Whoops! I did some mistake while rebasing the branch. I'll try to resolve this. Sorry!

**Comment by vatsalj** [comment]
I think I'll have to start over with my repository. This is getting too complicated. :-/ Thankfully, I always make a backup so work is not lost, yet.

**Pull request closed without merging**